boards/posix/native_posix/Kconfig.defconfig

```diff
@@ -15,6 +15,12 @@ config BOARD
 	default "native_posix_64" if BOARD_NATIVE_POSIX_64BIT
 	default "native_posix"
 
+config DISPLAY
+	default y if LVGL
+
+config KSCAN
+	default y if LVGL
+
```

Comment:
This seems a bit wrong to me.
Normally, here in the board defconfig you set default options for the board. So if, for example, a type of peripheral is enabled (say SERIAL), you enable that board default serial driver (UART_NATIVE_POSIX), so the user has a reasonable default configuration when changing target.

But what you are doing here is enabling a peripheral if a library is enabled. This does not fit in the board configuration. If a library needs a peripheral, then it would be a matter of setting in that library kconfig file a `depends on`, or in the worst case a `select`. (So LVGL could have a `depends on DISPLAY` , and/or DISPLAY could have a `default y if LVGL` in drivers/display/Kconfig)
The library kconfig dependencies should not change depending on the target board.

(On the other hand
```
if KSCAN
config KSCAN_SDL
	default y
```
Is the normal thing to do
)

Reply:
Agree here and LVGL should never be selected by default if DISPLAY and/or KSCAN is enabled.

Reply:
This more like a question. I wasn't sure would it be valid to do in such way. Thank you for explanation. Removed

Reply:
@jfischer-phytec-iot could you be so kind to clarify options on "LVGL and CONFIG_LVGL_POINTER_KSCAN". For me both points are making sense:
1) Enable by default (but some case may need no Display, or display without Kscan, etc)
2) Enable in each application (copy-paste)